apps/api/src/app/workflows/usecases/get-active-integrations-status/get-active-integrations-status.usecase.ts

```diff
@@ -113,7 +113,10 @@ export class GetActiveIntegrationsStatus {
     for (const step of uniqueSteps) {
       const stepType = step.template?.type;
       const skipStep =
-        stepType === StepTypeEnum.DELAY || stepType === StepTypeEnum.DIGEST || stepType === StepTypeEnum.TRIGGER;
+        stepType === StepTypeEnum.DELAY ||
+        stepType === StepTypeEnum.DIGEST ||
+        stepType === StepTypeEnum.TRIGGER ||
+        stepType === StepTypeEnum.CUSTOM;
```

Comment:
Create ActionStepTypeEnum and skip by it

```suggestion
const skipStep = ActionStepTypeEnum.values.includes((action) => action === stepType)
```